auth-web/src/views/auth/MaintainBusinessView.vue

```diff
@@ -36,6 +36,50 @@
           <learn-more-button
            :redirect-url="learnMoreUrl"
           />
+          <p v-if="!userProfile" class="mt-5">Don't have your Cooperative Passcode?
```

Comment:
It feels like this should be a sub-component.

Reply:
Originally had planned on it, but it was preferred by others to be a popover built into this component.

Reply:
ok